### ISSUE TITLE
Issues with next version of ggplot2

### ISSUE THREAD

**Issue by thomasp85** [great-northern-diver/ggmulti#15]
Hi

We are preparing the next release of ggplot2 and our tests show that ggmulti fails with the new version. Since you are doing some very unsafe overwriting of different internal ggplot2 functions it is difficult to say exactly where the issue is located.

I urge you to reconsider your current approach since it has the potential to break again and again, while also potentially break other packages in hard-to-debug ways because you are changing ggplot2 functionality that other packages depends on

we are planning to release the next version of ggplot2 on May 31st

best
Thomas

**Comment by z267xu** [comment]
Hi Thomas
Nice to meet you! 

For the developed version of the `ggplot2` from github (which is 3.3.3.9000), many tests passed but some tests depending on the function `ggplot2::flipped_names()` failed.  As I looked deep into it, I suppose there is a bug. Call `ggplot2::flipped_names(TRUE)`, we will get 

```
> packageVersion("ggplot2")
[1] '3.3.3.9000'
> ggplot2::flipped_names(TRUE)
... ...
$y
[1] "intercept"
$ymin
[1] "x"
$ymax
[1] "xmin"
$yend
[1] "xmax"
$yintercept
[1] "xend"
$ymin_final
[1] "xintercept"
$ymax_final
[1] "xmin_final"
$lower
[1] "xmax_final"
$middle
[1] "xlower"
$upper
[1] "xmiddle"
$y0
[1] "xupper"
$intercept
[1] "x0"
```
Looks like the orders of `y` flipped names are messed up here. 

Glad to hear back from you!

Best

**Comment by thomasp85** [comment]
Thanks for flagging this - this is indeed a bug that will get fixes before release

**Comment by thomasp85** [comment]
Should be fixed in the v3.3.4-rc branch now. Thanks!
